Lib/test/test_asyncio/test_futures2.py

```diff
@@ -1,9 +1,32 @@
 # IsolatedAsyncioTestCase based tests
 import asyncio
 import unittest
-
+import traceback
 
 class FutureTests(unittest.IsolatedAsyncioTestCase):
+    async def test_future_traceback(self): 
+        async def raise_exc():
+            raise TypeError(42)
+        future = asyncio.create_task(raise_exc())
+        for _ in range(10):
+            try:
+                await future
+            except TypeError:
```

Comment:
Please, add `else: self.fail('TypeError not raised')`

Reply:
Done